### ISSUE TITLE
Move user avatars to the Media Library

### ISSUE THREAD

**Issue by mundschenk-at** [mundschenk-at/avatar-privacy#75]
- Store user avatars and custom default icons in the Media Library.
- Migrate legacy images to the Media Library.

**Comment by dshanske** [comment]
For this, would they be stored in the dated directories or separately?

**Comment by mundschenk-at** [comment]
Not sure. You would not directly access them, so I don't think it would make much difference for the user/visitor, no?

**Comment by dshanske** [comment]
Yes, my issue is I share them and want the URL to be unchanging

**Comment by mundschenk-at** [comment]
Yes but the URL of the actual avatar image is independent of the URL/filepath used to store the uploaded image. (I don't think this is relevant to #80.)
